### PR TITLE
Update third party notices for Components and ProjectTemplates

### LINKED ISSUE #48727: Update Third Party Notices if needed
We need to check if Third Party Notices need to be updated.

https://github.com/dotnet/aspnetcore/blob/main/THIRD-PARTY-NOTICES.txt 
https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/THIRD-PARTY-NOTICES
https://github.com/dotnet/aspnetcore/blob/main/src/Components/THIRD-PARTY-NOTICES.txt

### PR DESCRIPTION
**Pull request by surayya-MS**
# Update third party notices for Components and ProjectTemplates

## Description

1. Removed Open Iconic license in Components third party notices and added Bootstrap Icons license 
2. Removed Angular license in ProjectTemplates third party notices because we removed SPA templates

Fixes #48727

## Customer Impact

1. Third party notices for Components and ProjectTemplates are updated

## Regression?

- [ ] Yes
- [x] No

[If yes, specify the version the behavior has regressed from]

## Risk

- [ ] High
- [ ] Medium
- [x] Low

This is third party notices update, no code change.

## Verification

- [x] Manual (required)
- [ ] Automated

## Packaging changes reviewed?

- [ ] Yes
- [ ] No
- [x] N/A

----

## When servicing release/2.1

- [ ] Make necessary changes in eng/PatchConfig.props

### CODE REVIEW

File: src/Components/THIRD-PARTY-NOTICES.txt [hunk 1]
```diff
@@ -398,104 +398,39 @@ THE SOFTWARE.
 =========================================
 
 
-%% License notice for Open Iconic fonts
```
Comment (surayya-MS) [inline src/Components/THIRD-PARTY-NOTICES.txt:401]: We now use Bootstrap Icons instead

File: src/ProjectTemplates/THIRD-PARTY-NOTICES [hunk 2]
```diff
@@ -200,28 +200,3 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
-
-License notice for Angular v8.0
```
Comment (surayya-MS) [inline src/ProjectTemplates/THIRD-PARTY-NOTICES:204]: We removed spa templates

**Comment by mkArtakMSFT** [comment]
Thanks @surayya-MS. The changes in those two files look good, but I believe you have to also update https://github.com/dotnet/aspnetcore/blob/release/8.0/THIRD-PARTY-NOTICES.txt

**Comment by surayya-MS** [comment]
>Thanks @surayya-MS. The changes in those two files look good, but I believe you have to also update https://github.com/dotnet/aspnetcore/blob/release/8.0/THIRD-PARTY-NOTICES.txt

@mkArtakMSFT there is only angular license in that file. It looks like we use angular in https://github.com/dotnet/aspnetcore/tree/38b164b0c90bfff60ff17b5ca13d1e87a37050d4/src/Middleware/Spa still

File: src/Components/THIRD-PARTY-NOTICES.txt [hunk 3]
```diff
@@ -22,7 +22,7 @@ The actual third-party libraries or other resources may vary depending on the Bl
 2.	AngleSharp  (https://github.com/AngleSharp/AngleSharp)
 3.	SimpleJson (https://github.com/facebook-csharp-sdk/simple-json)
```
Comment (surayya-MS) [inline src/Components/THIRD-PARTY-NOTICES.txt:23]: I'm not sure if we still use it. @SteveSandersonMS, @javiercn , @MackinnonBuck do you know?

**Review by mkArtakMSFT — APPROVED** [review]
Thanks @surayya-MS

**Pull request merged**